src/guides/v2.3/test/unit/writing_testable_code.md

```diff
@@ -158,7 +158,7 @@ As a rule of thumb, try to keep methods to five or fewer lines of code.
 
 Functions should do only one thing and they should do it very well.
 
-Once you respect the [single responsibility principle][]{:target="_blank"}, you will know exactly what you are testing and your functions will be smaller and clearer.
+Once you respect the [single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle){:target="_blank"}, you will know exactly what you are testing and your functions will be smaller and clearer.
```

Comment:
Hi @konarshankar07. Thank you for your collaboration. As a rule, we don't need to provide the URL explicitly (inline) in the link if we have the URL in "Links definitions". 
After this change we have the URL in both places: "Link definitions" and inline which does not seem to be the best practice.